config/src/test/java/org/springframework/security/config/annotation/method/configuration/MethodSecurityService.java

```diff
@@ -50,7 +50,7 @@ public interface MethodSecurityService {
 	@PermitAll
 	String jsr250PermitAll();
 
-	@RolesAllowed("ADMIN")
+	@RolesAllowed({ "ADMIN", "USER" })
```

Comment:
Will you please separate this into a user method, like you did with the others? In this way, the test methods will read more consistently.